waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -399,3 +408,38 @@ genMiddleware spec =
               "importStatement" .= maybe "" fst maybeGlobalMidlewareConfigFnImports,
               "importAlias" .= globalMiddlewareConfigFnAlias
             ]
+
+depsRequiredByWebSockets :: AppSpec -> [AS.Dependency.Dependency]
+depsRequiredByWebSockets spec
+  | AS.WS.areWebSocketsUsed spec = AS.WS.serverDepsRequiredForWebSockets
+  | otherwise = []
+
+genWebSockets :: AppSpec -> Generator [FileDraft]
+genWebSockets spec
+  | AS.WS.areWebSocketsUsed spec =
+      sequence
+        [ genWebSocketTs spec
+        ]
+  | otherwise = return []
+
+genWebSocketTs :: AppSpec -> Generator FileDraft
```

Comment:
Quite a lot of web socket functions, should we extract them into separate module?

Second, why does the name here end with `Ts`, does that symbolize that it uses Typescript, or that these are Types?
If the former, I would question if we need that, I haven't seen us using it before. If latter, I would expand it to full name, `Types`.

Reply:
Aha so we did use it hm! Ok, whatever you like the bets! I would be ok with nixing it also.